src/games/kalambury/board/GameBoard.js

```diff
@@ -21,14 +35,29 @@ const GameBoard = ({
   canChangePhrase,
   ChangePhrase,
   t,
-}) => (
+}) => {
+
+  const [lastSuccess, setLastSuccess] = useState(false);
+  const previousUserGuesses = getUserActions(actions, playerID, "guess")
+
+  useEffect(() => {
```

Comment:
You can only see confetti when you are Guessing phrases, so it makes sense to move it to `GuessingBoard`

Reply:
and in `GuessingBoard` you already have all the logic for `lastSuccess`

Reply:
But the GuessingBoard unmounts when you start drawing. So if you guess correctly and then it's your turn to draw you won't get confetti, because your guessing board has already been replaced with drawing board

Reply:
hmm.. 🤔 that's right

Reply:
but let me check that

Reply:
@pastDexter I've tried to use it in GuessingBoard but it doesn't work

Reply:
yep, it doesnt work if you are next person to draw, like @Bombasarkadian  said